javascript/core/scope/hoisting-applies-only-to-variable-declarations-not-initializations.md

```diff
@@ -48,25 +23,38 @@ aspects:
 ---
 ## Content
 
-Variable declarations are moved to the top of the current scope, either the current function or script. This means that variables can be used before they are declared.
-```
+Variable declarations using `var` are moved to the top of the function scope (or global scope if outside of a function). One consequences of this is that variables can be used before they are declared.
```

Comment:
Should be "One consequence"

Reply:
Nice catch.

Give me a suggestion like this next time so that I can just click to change :D

```suggestion
Variable declarations using `var` are moved to the top of the function scope (or global scope if outside of a function). One consequence of this is that variables can be used before they are declared.
```